### PR TITLE
chore: add Fullstack Open

### LINKED ISSUE #729: [Add] Free Resource for React Js under frontend category
### Site URL

https://fullstackopen.com/en/about/

### Description of the above provided site

This course serves as an introduction to modern web application development with JavaScript. The main focus is on building single page applications with ReactJS that use REST APIs built with Node.js. The course also contains a section on GraphQL, a modern alternative to REST APIs.

### Category

Frontend

### Sub Category

React Js

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am a GSSoC'23 contributor

### Note

I have experience contributing to various projects in open-source, and I have chosen LinksHub to work on GSSoC'23 because of its high standards and guidelines, which give it credibility. 
Therefore, I kindly request permission to work on different issues simultaneously.

### PR DESCRIPTION
**Pull request by Anmol-Baranwal**
## Fixes Issue

closes #729 

## Changes proposed

- Add Fullstack Open in React under Frontend category

## Screenshots

image

## Note to reviewers

- I will resolve any merge conflicts that may arise.

### CODE REVIEW

File: database/frontend/react.json [hunk 1]
```diff
@@ -1,10 +1,18 @@
...
+        "name": "React js",
+        "description": "Free online course for learning react js",
+        "url": "https://scrimba.com/learn/learnreact#",
+        "category": "frontend",
+        "subcategory": "react",
+        "language": "english"
+    },
+    {
+        "name": "Fullstack Open",
+        "description": "This course serves as an introduction to modern web application development with JavaScript. The main focus is on building single page applications with ReactJS that use REST APIs built with Node.js. The course also contains a section on GraphQL, a modern alternative to REST APIs.",
```
Comment (CBID2) [inline database/frontend/react.json:12]: [Proofreading]
```suggestion
        "description": "This course serves as an introduction to modern web application development with JavaScript. The main focus is on building single page applications with ReactJS and use REST APIs built with Node.js. The course also contains a section on GraphQL, a modern alternative to REST APIs.",

```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @Anmol-Baranwal! :) I pointed out a typo in my inline comment. In the future, consider running your descriptions through Grammarly or Spellcheck before making a pull request. Other than that, you're off to a great start! :)

File: database/frontend/react.json (hunk 1 above)
Comment (Anmol-Baranwal) [reply database/frontend/react.json]: I have made the changes. Thank you for your assistance. In the future, I will ensure to use Grammarly to check for any grammar errors.

**Review by CBID2 — APPROVED** [review]
Great job @Anmol-Baranwal! 😄 LGTM! 😄

**Comment by rupali-codes** [comment]
@Anmol-Baranwal please resolve merge conflicts

File: database/frontend/react.json [hunk 2]
```diff
@@ -19,5 +19,13 @@
...
         "category":"frontend",
         "subcategory":"react"
+    },
+    {
+        "name": "Fullstack Open",
+        "description": "This course serves as an introduction to modern web application development with JavaScript. The main focus is on building single page applications with ReactJS and use REST APIs built with Node.js. The course also contains a section on GraphQL, a modern alternative to REST APIs.",
+        "url": "https://fullstackopen.com/en/",
+        "category": "frontend",
+        "subcategory": "react",
+        "language": "english"
```
Comment (rupali-codes) [inline database/frontend/react.json:29]:
```suggestion
```

File: database/frontend/react.json [hunk 3]
```diff
@@ -19,5 +19,13 @@
         "url":"https://react.dev/",
         "category":"frontend",
         "subcategory":"react"
+    },
+    {
+        "name": "Fullstack Open",
+        "description": "This course serves as an introduction to modern web application development with JavaScript. The main focus is on building single page applications with ReactJS and use REST APIs built with Node.js. The course also contains a section on GraphQL, a modern alternative to REST APIs.",
+        "url": "https://fullstackopen.com/en/",
+        "category": "frontend",
+        "subcategory": "react",
```
Comment (rupali-codes) [inline database/frontend/react.json:28]:
```suggestion
        "subcategory": "react"

```

**Comment by Anmol-Baranwal** [comment]
image

Since I use Prettier, the file is automatically formatted upon saving. Here is a screenshot of the modified file, reflecting the changes.

**Comment by rupali-codes** [comment]
Great, thanks

**Pull request merged**